### ISSUE TITLE
Meteor downgrade from 1.5 to 1.4.6

### ISSUE THREAD

**Issue by fireflypie** [meteor/meteor#8851]
Hello,

After an update from 1.4 Meteor to 1.5 I started to get this error

```
C:\Projects\conetrips>meteor update

Changes to your project's package version selections from updating the
release:

accounts-base              upgraded from 1.2.15 to 1.3.0
accounts-password          upgraded from 1.3.4 to 1.3.6
babel-compiler             upgraded from 6.14.1 to 6.19.1
boilerplate-generator      upgraded from 1.0.11 to 1.1.0
ddp-client                 upgraded from 1.3.3 to 1.3.4
ddp-server                 upgraded from 1.3.13 to 1.3.14
dynamic-import             added, version 0.1.0
ecmascript                 upgraded from 0.6.3 to 0.8.0
ecmascript-runtime         upgraded from 0.3.15 to 0.4.1
ecmascript-runtime-client  added, version 0.4.1
ecmascript-runtime-server  added, version 0.4.1
email                      upgraded from 1.1.18 to 1.2.1
localstorage               upgraded from 1.0.12 to 1.1.0
meteor-base                upgraded from 1.0.4 to 1.1.0
minifier-js*               upgraded from 1.2.18 to 2.1.0
minimongo                  upgraded from 1.0.21 to 1.2.0
modules                    upgraded from 0.7.9 to 0.9.0
modules-runtime            upgraded from 0.7.9 to 0.8.0
mongo                      upgraded from 1.1.16 to 1.1.18
promise                    upgraded from 0.8.8 to 0.8.9
rate-limit                 upgraded from 1.0.7 to 1.0.8
reactive-dict              upgraded from 1.1.8 to 1.1.9
standard-minifier-js*      upgraded from 1.2.3 to 2.1.0
tracker                    upgraded from 1.1.2 to 1.1.3
webapp                     upgraded from 1.3.14 to 1.3.16


* These packages have been updated to new versions that are not
backwards
compatible.
conetrips: updated to Meteor 1.5.

Changes to your project's package version selections from updating package
versions:

accounts-password          upgraded from 1.3.6 to 1.3.7
anychart:anychart          upgraded from 1.1.0 to 1.2.0
babel-compiler             upgraded from 6.19.1 to 6.19.3
dynamic-import             upgraded from 0.1.0 to 0.1.1
ecmascript                 upgraded from 0.8.0 to 0.8.1
ecmascript-runtime-client  upgraded from 0.4.1 to 0.4.2
email                      upgraded from 1.2.1 to 1.2.3
fourseven:scss             upgraded from 4.5.0 to 4.5.3
minimongo                  upgraded from 1.2.0 to 1.2.1
modules                    upgraded from 0.9.0 to 0.9.2
npm-bcrypt                 upgraded from 0.9.2 to 0.9.3


C:\Projects\conetrips>meteor
[[[[[ C:\Projects\conetrips ]]]]]

=> Started proxy.
=> Started MongoDB.
C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.windows.x86
_32\dev_bundle\lib\node_modules\fibers\future.js:313
                                                throw(ex);
                                                ^

Error: ENOTEMPTY: directory not empty, rmdir 'C:\Projects\conetrips\.meteor\loca
l\build-garbage-1jhdet5\programs\server\assets\packages\boilerplate-generator'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:332:11)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at rimrafSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:293:9)
    at C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.wind
ows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:330:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:329:26)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at rimrafSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:293:9)
    at C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.wind
ows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:330:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:329:26)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at rimrafSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:293:9)
    at C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.wind
ows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:330:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:329:26)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at rimrafSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:293:9)
    at C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.wind
ows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:330:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:329:26)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at rimrafSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:293:9)
    at C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0\mt-os.wind
ows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:330:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.
0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:329:26)
    at rmdirSync (C:\Users\Alex\AppData\Local\.meteor\packages\meteor-tool\1.5.0
\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rimraf.js:322:7)
    at Function.rimrafSync [as sync] (C:\Users\Alex\AppData\Local\.meteor\packag
es\meteor-tool\1.5.0\mt-os.windows.x86_32\dev_bundle\lib\node_modules\rimraf\rim
raf.js:293:9)
    at Object.rm_recursive (C:\tools\fs\files.js:303:12)
    at Object.renameDirAlmostAtomically (C:\tools\fs\files.js:1041:13)
    at Builder.complete (C:\tools\isobuild\builder.js:690:13)
    at C:\tools\isobuild\bundler.js:2719:13
    at C:\tools\isobuild\bundler.js:3027:22
    at C:\tools\utils\buildmessage.js:271:13
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14
)
    at C:\tools\utils\buildmessage.js:264:29
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14
)
    at C:\tools\utils\buildmessage.js:262:18
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14
)
    at C:\tools\utils\buildmessage.js:253:23
    at [object Object]._.extend.withValue (C:\tools\utils\fiber-helpers.js:89:14
)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at bundle (C:\tools\isobuild\bundler.js:2860:31)
    at C:\tools\isobuild\bundler.js:2807:32
    at Object.withCache (C:\tools\fs\files.js:1652:12)
    at Object.exports.bundle (C:\tools\isobuild\bundler.js:2807:16)
    at C:\tools\runners\run-app.js:588:36
    at Function.run (C:\tools\tool-env\profile.js:490:12)
    at bundleApp (C:\tools\runners\run-app.js:587:34)
    at [object Object]._.extend._runOnce (C:\tools\runners\run-app.js:631:35)
    at [object Object]._.extend._fiber (C:\tools\runners\run-app.js:890:28)
    at C:\tools\runners\run-app.js:417:12

C:\Projects\conetrips>meteor
[[[[[ C:\Projects\conetrips ]]]]]

=> Started proxy.
=> Errors prevented startup:

   While processing files with fourseven:scss (for target web.browser):
   /packages/themeteorchef_bert/stylesheets/bert.scss: Scss compiler
   error: _.without is not a function


=> Your application has errors. Waiting for file change.
=> Started MongoDB.
```

How do I rollback changes? Or what to do this case when there's an error caused by a vital plugin.

**Comment by hwillson** [comment]
Just to add - the original `_.without` issue in `themeteorchef:bert` is caused by the `fourseven:scss` package. Version 4.5.4 of `fourseven:scss` fixes this issue (see https://github.com/fourseven/meteor-scss/issues/252), so as soon as `themeteorchef:bert` is updated to use 4.5.4, this issue should be fixed.

Let us know if you're still having `rimraf` issues after following the suggestion in https://github.com/meteor/meteor/issues/8851#issuecomment-311356206. Closing for now - thanks!

**Comment by hwillson** [comment]
One more addition - the `_.without` issue was just fixed in `themeteorchef:bert` version `2.1.3` (see https://github.com/themeteorchef/bert/issues/51#event-1146489959).
